### DISCUSSION TITLE
Unable to create relationships in GUI from exisiting data

### DISCUSSION THREAD

**Discussion by accesstechnology-mike** [directus/directus#17232] [Archive]
Relationship option not displaying when configuring Database only fields:

![image](https://user-images.githubusercontent.com/55278705/213566942-e891101b-553a-46a3-bc6c-f9e0c1351a2f.png)

Manually editing directus_relations table does then show the (non-editable) option:

![image](https://user-images.githubusercontent.com/55278705/213567271-259b8477-d4bf-4238-b3b8-ccd327f9736e.png)

**Comment by br41nslug** [comment]
It's not directly clear to me where you'd like to display this but that's exactly what the "display templates" are meant to do in various places 
![image](https://user-images.githubusercontent.com/9389634/213444820-c915129f-2706-41d7-bc57-202608a362ed.png)

**Comment by accesstechnology-mike** [comment]
Ah, then there's an issue with my setup. 

Is there a common reason why no relationally linked fields show up?

**Reply by br41nslug** [reply]
How have you linked the fields? They should show up if you've created relational fields.

**Reply by accesstechnology-mike** [reply]
They're linked as foreign keys in sql database. Do I need to link them within directus?

**Reply by br41nslug** [reply]
> Do I need to link them within directus?

Yes the relations need to be known to Directus to traverse them 😄 
https://docs.directus.io/configuration/data-model/relationships.html
